specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json

```diff
@@ -921,6 +921,50 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/rotateClusterCertificates": {
+      "post": {
+        "tags": [
+          "ManagedClusters"
+        ],
+        "operationId": "ManagedClusters_RotateClusterCertificates",
+        "summary": "Rotate certificates of a managed cluster.",
+        "description": "Rotate certificates of a managed cluster.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
```

Comment:
Is there no response body to the action?

Reply:
I have removed 200 response, there is only 202

Reply:
now I use 200 and 204 instead.
```
A 'POST' operation 'ManagedClusters_RotateClusterCertificates' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201 or 204
```